src/app/docs/docs.component.html

```diff
@@ -1,7 +1,7 @@
 <div class="docs">
   <elements-stoplight-project
-    [workspace]="workspace"
-    [project]="project"
+    [workspaceSlug]="workspaceSlug"
+    [projectSlug]="projectSlug"
```

Comment:
Interesting, I did the exact same thing and it failed on my machine. Nonetheless, thanks for updating that!

Reply:
I did. Error was returned, telling me that /docs could not be found. Although when I pull your branch now, everything seems to work, so maybe I did something different